src/components/PreviewOverlay.tsx

```diff
@@ -1,81 +1,90 @@
 import styled from "styled-components";
 import React, { useContext } from "react";
 import useKeypress from "react-use-keypress";
+import { IconButton } from "@mui/material";
+import ArrowBackSharpIcon from "@mui/icons-material/ArrowBackSharp";
 import TextPreview from "./Editor/TextPreview";
 import { useSelected } from "../util/customHooks/previewOverlayfunc";
 import { ActionData } from "../util/types";
+import { defaultCameraView } from "../util/unityContextActions";
 import { SideBarContext } from "../util/customHooks/SideBarContext";
 
-const PreviewRoot = styled.div`
-  display: absolute;
-  width: inherit;
-  height: 750px;
-  z-index: 0;
-  opacity: 1;
-  pointer-events: auto;
-`;
-
 const PreviewGrid = styled.div`
   display: grid;
   width: inherit;
   height: inherit;
   grid-template-columns: repeat(60, 1fr);
-  grid-template-rows: repeat(60, 1fr);
+  grid-template-rows: repeat(50, 1fr);
 `;
 
+const Header = styled.div`
+  position: relative;
+  grid-column: 1 / span 58;
+  grid-row: 1 / 2;
+  z-index: 2;
+  pointer-events: auto;
+`;
 const PreviewGridCenter = styled.div`
-  grid-column: 21 / span 20;
-  grid-row: 50 / span 15;
+  position: relative;
+  grid-column: 2 / span 58;
+  grid-row: 2 / span 47;
   z-index: 2;
   pointer-events: auto;
-  min-height: 1em;
-  min-width: inherit;
-  padding-left: 5%;
-  padding-right: 5%;
 `;
 
 type PreviewOverlayProps = {
   actionList: ActionData[];
+  toggle: () => void;
 };
 
-// TODO change styling instead of defining a new component
-function PreviewOverlay({ actionList }: PreviewOverlayProps): JSX.Element {
+function PreviewOverlay({
+  actionList,
+  toggle,
+}: PreviewOverlayProps): JSX.Element {
```

Comment:
maybe add these parts into the global constant?

Reply:
Yes that's a very good point. I will fix this